src/app/pages/profile/personal-information/personal-information.component.ts

```diff
@@ -30,28 +30,28 @@ export class PersonalInformationComponent implements OnInit, OnDestroy {
   };
   profileForm: FormGroup;
   firstName = new FormControl(
-    { value: '', disabled: true },
+    { value: "", disabled: true },
     Validators.required
   );
   lastName = new FormControl(
-    { value: '', disabled: true },
+    { value: "", disabled: true },
     Validators.required
   );
-  emailAddress = new FormControl({ value: '', disabled: true }, [
+  emailAddress = new FormControl({ value: "", disabled: true }, [
     Validators.required,
     Validators.email
   ]);
-  street = new FormControl('', Validators.required);
-  city = new FormControl('', Validators.required);
-  state = new FormControl('', Validators.required);
-  phone = new FormControl('', [Validators.required, Validators.minLength(14)]);
-  employer = new FormControl('');
-  designation = new FormControl('');
+  street = new FormControl("", Validators.required);
```

Comment:
Could you refactor your code to take advantage of the FormGroup() provided by Angular? If you need a sync on this let me know.

Reply:
Hey @malep2007 , thankyou for the feedback but this was not part of my feature. I think when I configured my linter, it then reflects that I changed most parts of the code